internal/auth/providers/google_admin.go

```diff
@@ -52,7 +52,7 @@ func getAdminService(adminEmail string, credentialsReader io.Reader) *admin.Serv
 }
 
 // GetMembers returns the members of a google group
-func (gs *GoogleAdminService) GetMembers(groupName string) ([]string, error) {
+func (gs *GoogleAdminService) GetMembers(groupName string, tail int) ([]string, error) {
```

Comment:
Super minor, but it's not obvious to me why this is called `tail`.  Seems like either `depth` (where we increment on recursion) or `maxDepth` (decrement) would be clearer.